SCP -> BTP, Cloud Foundry - Cloud Foundry environment (#199)
SAP/cloud-sdk: docs/js/faq.mdx

## Questions About SAP Business Technology Platform

> ### Do You Support SAP Business Technology Platform - Cloud Foundry environment?

Author: ```suggestion
### Do You Support SAP Business Technology Platform - Cloud Foundry Environment?
```

> ### Do You Support SAP Business Technology Platform - Cloud Foundry Environment?

Reviewer: ```suggestion
### Do You Support SAP Business Technology Platform, Cloud Foundry Environment?
```

> SAP Cloud SDK for JavaScript has first-class support for [SCP Cloud Foundry environment](https://www.sap.com/products/cloud-platform.html).

Reviewer: ```suggestion
SAP Cloud SDK for JavaScript has first-class support for the [SAP BTP, Cloud Foundry environment](https://www.sap.com/products/cloud-platform.html).
```

We provide plenty of helpful abstractions for [connectivity](features/connectivity/destination) and authentication that make developing apps a pleasant and rewarding experience.

> Let us know if you're missing any features about BTP Cloud Foundry environment support from SAP Cloud SDK for JavaScript.

Reviewer: ```suggestion
Let us know if you miss functionality of the SAP BTP in the SAP Cloud SDK for JavaScript.
```
